[ECOINT-111] Add Inngest integration (#2660)
DataDog/integrations-extras: inngest/README.md

# Inngest

## Overview

Send key metrics about your Inngest functions directly to your Datadog account.

## Setup

> Install this integration by clicking "Connect Accounts" or "Install Integration" above. You may be asked to sign into your Inngest account, and then you'll be asked to authorize Inngest to access your Datadog organization.

Reviewer: Since this content appears on a product docs page as well as an in-app integration tile, directions like "above" don't apply to all contexts.
```suggestion
Install this integration by clicking **Connect Accounts** or **Install Integration**. You may be asked to sign into your Inngest account and authorize Inngest to access your Datadog organization.
```

> After you authorize Inngest, your Production Inngest environment will begin sending metrics to Datadog. You can enable additional environments to send metrics in your Inngest Cloud dashboard → Integrations → Datadog.

Reviewer: ```suggestion
After you authorize Inngest, your Production Inngest environment will begin sending metrics to Datadog. To enable additional environments to send metrics, go to your Inngest Cloud dashboard > **Integrations** > **Datadog**.
```

## Uninstallation

> To remove this integration, navigate to your Inngest Cloud dashboard → Integrations → Datadog. Click the Remove button next to each Datadog organization you wish to disconnect from Inngest.

Reviewer: ```suggestion
To remove this integration, navigate to your Inngest Cloud dashboard > **Integrations** > **Datadog**. Click **Remove** next to each Datadog organization you wish to disconnect from Inngest.
```

## Support

> [Chat with us on Discord][1] or [create a support ticket][2].

Reviewer: ```suggestion
[Chat with Inngest on Discord][1] or [create a support ticket][2].
```

# Inngest

## Overview

Inngest's durable functions replace queues, state management, and scheduling to enable developers to build reliable, AI-ready step functions faster - without touching infrastructure.

> Send key metrics about your Inngest functions and their underlying steps directly to your Datadog account. This allows you to proactively identify errors or performance issues, right alongside the monitoring you use for the rest of your infrastructure.

Reviewer 2: ```suggestion
Send key metrics about your Inngest functions and their underlying steps directly to your Datadog account. This allows you to proactively identify errors or performance issues, in addition to the monitoring you use for the rest of your infrastructure.
```

## Setup

1. In Datadog, navigate to **Integrations**, select the Inngest tile, and click **Install Integration**.

> 2. Click **Connect Accounts** to start the authorization process. You will be redirected to Inngest to complete the setup.

Reviewer 2: ```suggestion
2. Click **Connect Accounts** to start the authorization process. You are redirected to Inngest to complete the setup.
```
